### PR TITLE
Update Timestamp to ISO9601

### PR DESCRIPTION
**Pull request by knockknockyoo**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const {DATABASE_NAME, APPLICATION_COLLECTION, USER_COLLECTION, ORGANIZATION_COLL
 const {Application} = require("./services/application");
 const {MongoQueries} = require("./crdc-datahub-database-drivers/mongo-queries");
 const {DatabaseConnector} = require("./crdc-datahub-database-drivers/database-connector");
-const {getCurrentTimeYYYYMMDDSS} = require("./utility/time-utility");
+const {getCurrentTimeYYYYMMDDSS} = require("./crdc-datahub-database-drivers/utility/time-utility");
```
Comment (n2iw) [inline app.js:17]: Function needs a better name, see PR comment in submodule.

File: services/organization.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
     }
 
     async getOrganizationByID(id) {
         let result = await this.organizationCollection.find(id);
-        return result?.length > 0 ? result[0] : null;
+        return result?.length > 0 ? toISOTime(result[0]) : null;
     }
 }
 
+const toISOTime = (aOrg) => {
```
Comment (n2iw) [inline services/organization.js:13]: This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

File: services/application.js [hunk 3]
```diff
@@ -363,6 +360,18 @@ function verifyReviewerPermission(context){
         .verifyRole([ROLES.ADMIN, ROLES.FEDERAL_LEAD]);
 }
 
+const toISOTime = (aApp) => {
```
Comment (n2iw) [inline services/application.js:363]: This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

File: services/application.js [hunk 4]
```diff
@@ -209,7 +206,7 @@ class Application {
         const deletedOne = await this.getApplicationById(document._id);
         let result = null;
         if (deletedOne && await this.dbService.deleteOne(APPLICATION, {_id: document._id})) {
-            result = deletedOne[0];
```
Comment (n2iw) [inline services/application.js:212]: Is this change from **deleteOne[0]** to **deleteOne** intentional?

Also, we should NOT delete any application records from DB, should only change status to "Deleted" instead.

**Review by n2iw — CHANGES_REQUESTED** [review]
Please see individual comments.

File: app.js (hunk 1 above)
Comment (knockknockyoo) [reply app.js]: updated!

File: services/organization.js (hunk 2 above)
Comment (knockknockyoo) [reply services/organization.js]: updated!

File: services/application.js (hunk 3 above)
Comment (knockknockyoo) [reply services/application.js]: updated!

File: services/application.js (hunk 4 above)
Comment (knockknockyoo) [reply services/application.js:212]: It was on our to-do list. I have now pushed it for an update.

File: crdc-datahub-database-drivers
Comment (n2iw) [inline crdc-datahub-database-drivers:1]: PR in submodule has been merged to 1.0.0 branch, please update submodule to the latest 1.0.0, thanks!

File: services/application.js (hunk 4 above)
Comment (n2iw) [reply services/application.js:212]: deleteApplication function needs permission control, please add a TODO comment in the function, so we won't forget about it.

**Review by n2iw — CHANGES_REQUESTED** [review]
Please see two new comments, thanks!

**Comment by knockknockyoo** [comment]
> 

Thank you. I've added a TODO comment.

**Pull request merged**